code/modules/client/preferences_toggles.dm

```diff
@@ -195,6 +195,13 @@
 	to_chat(usr, "You will [(prefs.chat_toggles & CHAT_BANKCARD) ? "now" : "no longer"] be notified when you get paid.")
 	SSblackbox.record_feedback("nested tally", "preferences_verb", 1, list("Toggle Income Notifications", "[(prefs.chat_toggles & CHAT_BANKCARD) ? "Enabled" : "Disabled"]"))
 
+/client/verb/toggle_eorg_arena()
+	set name = "Toggle Post-Round Arena"
+	set category = "Preferences"
+	set desc = "Toggles your participation in the Post-Round Arena"
+	prefs.eorg_arena = !prefs.eorg_arena
+	to_chat(usr, "You will [(prefs.eorg_arena) ? "now" : "no longer"] take part in the Post-Round Arena.)")
```

Comment:
```suggestion
	to_chat(usr, "You will [(prefs.eorg_arena) ? "now" : "no longer"] take part in the Post-Round Arena.")
```
Currently, it writes to chat like this: 
![grafik](https://user-images.githubusercontent.com/83819203/127755954-e418edcc-170a-463b-9153-fcf95a130dc0.png)